src/app/functions/server/onerep.ts

```diff
@@ -127,7 +126,7 @@ export async function createProfile(
     body: JSON.stringify(requestBody),
   });
   if (!response.ok) {
-    log.info(
+    logger.info(
```

Comment:
We might want to change this to error or warning?

Reply:
Hm yeah these are definitely errors, thanks